ui/app/rif/actions.js

```diff
@@ -73,4 +85,21 @@ function checkDomainAvailable (domainName) {
   }
 }
 
+function getDomainDetails (domainName) {
+  return (dispatch) => {
+    dispatch(actions.showLoadingIndication())
+    return new Promise((resolve, reject) => {
+        background.rif.rns.resolver.getDomainDetails(domainName, (error, details) => {
+          console.log("This are the details bringed", details)
```

Comment:
Can we use console.debug for those cases?

Reply:
Changed all console.log to .debug, thnks